subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/dependencies/AbstractExternalModuleDependency.java

```diff
@@ -82,6 +83,9 @@ public boolean isForce() {
     @Override
     public ExternalModuleDependency setForce(boolean force) {
         validateMutation(this.force, force);
+        if (force) {
+            DeprecationLogger.nagUserOfDeprecatedThing("Using force on a dependency is not recommended", "Consider using strict version constraints instead (version { strictly ... } })");
```

Comment:
This will render as `Using force on a dependency is not recommended This has been deprecated and is scheduled to be removed in Gradle 7.0. Consider using strict version constraints instead (version { strictly ... } })`

:x: There should be a _dot_ at the end of each sentence:
`DeprecationLogger.nagUserOfDeprecatedThing("Using force on a dependency is not recommended.", "Consider using strict version constraints instead (version { strictly ... } }).");
`

We don't necessarily need an integration test fo this (but you could consider adding one). Usually adding one lets you see these kind of "annoyances". And many users will see this message.